playbooks/common/openshift-node/config.yml

```diff
@@ -173,6 +173,15 @@
       dest: "{{ node_cert_dir }}"
     when: certs_missing
 
+- name: Configure node dnsmasq
+  hosts: oo_nodes_to_config:!oo_containerized_master_nodes
+  vars:
+    dns_domain: "{{ openshift.common.dns_domain }}"
+    kube_svc_ip: "{{ hostvars[groups.oo_first_master.0].openshift.master.kube_svc_ip }}"
+    portal_net: "{{ openshift.common.portal_net }}"
```

Comment:
Since these are stored in openshift_facts, we should just add a dependency for openshift_facts in openshift_node_dns

We probably want to store the kube_svc_ip var in openshift_facts for all hosts as well. Maybe cluster_dns_ip?